types/aws-lambda/test/cognito-tests.ts

```diff
@@ -271,3 +272,19 @@ const customEmailSender: CustomEmailSenderTriggerHandler = async (event, _, call
     triggerSource === 'CustomEmailSender_SignUp';
     triggerSource === 'CustomEmailSender_AccountTakeOverNotification';
 };
+
+const customSmsSender: CustomSMSSenderTriggerHandler = async (event, _, callback) => {
+    const { request, response, triggerSource } = event;
+
+    str = request.type;
+    strOrNull = request.code;
+    obj = request.userAttributes;
+    objectOrUndefined = request.clientMetadata;
+
+    triggerSource === 'CustomSMSSender_AdminCreateUser';
+    triggerSource === 'CustomSMSSender_VerifyUserAttribute';
+    triggerSource === 'CustomSMSSender_UpdateUserAttribute';
+    triggerSource === 'CustomSMSSender_ResendCode';
+    triggerSource === 'CustomSMSSender_SignUp';
+    triggerSource === 'CustomSMSSender_Authentication';
```

Comment:
Small nit, this is missing `CustomSMSSender_ForgotPassword`

Reply:
thank you. Fixed.